devito/ir/iet/scheduler.py

```diff
@@ -80,13 +81,15 @@ def _lower_stepping_dims(iet):
         # two different calls to `xreplace`
         mindices = [d for d in i.uindices if d.is_Modulo]
         groups = as_mapper(mindices, lambda d: d.modulo)
+        i_org = i
```

Comment:
Nitpicking: throughout Devito, to identify and refer to the root of an IET we typically use the variable name "root" ie `root = i`

Reply:
Just to make sure: Even if `i` does not refer to the root of `iet`? For example we have `iet` argument, so we would use root to refer to these iterations?
```
<List (0, 3, 0)>
  <[affine,sequential] Iteration time::time::(time_m, time_M, 1)>
  	...
  <TimedList (2, 1, 2)>
	...
  <[affine,sequential] Iteration time::time::(time_m, time_M, 1)>
  	...
```